sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/feedranges/FeedRangeInternal.java

```diff
@@ -115,4 +302,47 @@ public static FeedRangeInternal tryParse(final String jsonString) {
             return null;
         }
     }
+
+    private static long fromHexEncodedBinaryString(String hexBinary)
+    {
```

Comment:
nit:
java code style is to have "{" on the same line as method definition, if, while, etc. here and elsewhere in this method.

Reply:
Fixed in next iteration